### PR TITLE
feat: implement error boundary

### PR DESCRIPTION
**Pull request by alter-eggo**
This pr implements global error boundary

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,6 +72,7 @@
     "pluralize": "^8.0.0",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-error-boundary": "^4.0.13",
```
Comment (garethfuller) [inline package.json:75]: Why is this required? Is it not all provided by Next.js?

**Review by garethfuller — COMMENTED** [review]
Can you add some description on how to test this so we can see all the ways this can be displayed?

**Review by agualis — APPROVED** [review]

**Review by garethfuller — CHANGES_REQUESTED** [review]
Why does it reload twice?

Screenshot 2024-08-30 at 14 58 37
